op-e2e/e2eutils/retryproxy/proxy.go

```diff
@@ -162,6 +162,7 @@ func (p *RetryProxy) doProxyReq(ctx context.Context, body []byte) (*http.Respons
 	if err != nil {
 		panic(fmt.Errorf("failed to create request: %w", err))
 	}
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
Why doesn't upstream have this issue, are we using the same l1 client as upstream?

Reply:
It could be the different implementation/configuration on the L1 clients.